cli/tools/mod.rs

```diff
@@ -7,5 +7,5 @@ pub mod installer;
 pub mod lint;
 pub mod repl;
 pub mod standalone;
-pub mod test_runner;
+pub mod test;
```

Comment:
Why change the module name? I don't think it's necessary - actually `test_runner` is more descriptive than `test`